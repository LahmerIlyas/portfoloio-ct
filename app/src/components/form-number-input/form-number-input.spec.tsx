import React from 'react';
import renderer from 'react-test-renderer';
import { Formik } from 'formik';
import { FormNumberInput } from '.';

describe('<FormNumberInput />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Formik initialValues={{}} onSubmit={() => {}}>
          <FormNumberInput field="size" title="Size" />
        </Formik>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
