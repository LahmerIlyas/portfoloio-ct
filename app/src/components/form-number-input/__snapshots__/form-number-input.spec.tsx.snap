// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormNumberInput /> renders correctly 1`] = `
<View
  style={
    Object {
      "marginVertical": 16,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontFamily": "Signika-Regular",
        "fontSize": 18,
      }
    }
  >
    Size
  </Text>
  <TextInput
    allowFontScaling={true}
    keyboardType="numeric"
    onChangeText={[Function]}
    rejectResponderTermination={true}
    style={
      Object {
        "backgroundColor": "#F4F4F4",
        "borderRadius": 8,
        "color": "#666666",
        "fontFamily": "Signika-Regular",
        "fontSize": 16,
        "marginTop": 8,
        "paddingHorizontal": 24,
        "paddingVertical": 24,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
